guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -305,6 +307,8 @@ The following example adds the `date_closed` column.
 </schema>
 ```
 
+When adding a new field into table, remember to [generate]({{ page.baseurl}}/extension-dev-guide/declarative-schema/migration-commands.html#create-whitelist) the `db_schema_whitelist.json` file.
```

Comment:
Should this be "column" instead of field? 

If yes, change to "When adding a column to a table"
If no, change to "When adding a field"

Reply:
> If yes, change to "When adding a column to a table"

**fixed**